brave/brave-core
# feat(wallet): Update Swap Setting Modal

## Description 
Updates the `Swap Settings` modal to the new design


Resolves <https://github.com/brave/brave-browser/issues/34582>



## Submitter Checklist:

  * `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests` [wiki](https://github.com/brave/brave-browser/wiki/Tests)
  * `npm run lint`, `npm run presubmit` [wiki](https://github.com/brave/brave-browser/wiki/Presubmit-checks), `npm run gn_check`, `npm run tslint`

## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:
1. Open the `Wallet` and go to the `Swap` screen
2. Open the `Swap Settings`, you should see the updated design
3. Settings should continue to work without any regressions.

Before:

![Screenshot 15](https://github.com/brave/brave-core/assets/40611140/56b4f686-66b9-476f-81e7-8f8190adfa45)

After:

![Screenshot 14](https://github.com/brave/brave-core/assets/40611140/d835c1c8-0f26-4431-ab98-56a5ae1590c5)
